test/sds/SDS_Regression.ipynb

```diff
@@ -878,6 +891,100 @@
+    "\n",
+    "This request will combine a variable and temporal subset for a granule in the M2T1NXSLV collection (MERRA-2). The result will include the requested variable and the three associated dimension variables:\n",
+    "\n",
+    "* `/H1000`\n",
+    "* `/lat`\n",
+    "* `/lon`\n",
+    "* `/time`\n",
+    "\n",
+    "Furthermore, the temporal dimension and the science variable (`/H1000`) will be limited to the specified temporal range. MERRA-2 is gridded at hourly intervals, so the 4-hour time range (12pm - 4pm on 9th June 2021) will return 4 time values.\n",
+    "\n",
+    "For the granule being tested, all time values are expressed as minutes since 2021-06-09T00:30:00Z, and each pixel spans 30 minutes in either direction from the stated value (cells are centre-aligned). As such:\n",
+    "\n",
+    "* 12pm is the leading edge of the cell centred at 12:30pm, which is 720 minutes since 00:30am.\n",
+    "* 4pm is the trailing edge of the cell centred at 15:30pm, which is 900 minutes since 00:30am."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c9e495e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if hoss_info is not None:\n",
+    "    hoss_temporal_file_name = 'hoss_temporal.nc4'\n",
+    "    hoss_temporal_request = Request(collection=hoss_info['temporal_collection'],\n",
+    "                                    granule_id=hoss_info['temporal_granule_id'],\n",
+    "                                    variables=['H1000'],\n",
+    "                                    temporal={'start': datetime(2021, 6, 9, 12, 0, 0),\n",
+    "                                              'stop': datetime(2021, 6, 9, 16, 0, 0)})\n",
+    "\n",
+    "\n",
+    "    submit_and_download(harmony_client, hoss_temporal_request, hoss_temporal_file_name)\n",
+    "    assert exists(all_no_bbox_file_name), 'Unsuccessful HOSS temporal request.'\n",
+    "\n",
+    "    expected_variables = ['/H1000', '/lat', '/lon', 'time']\n",
+    "    assert all_variables_present(hoss_temporal_file_name, expected_variables), 'Missing variables in HOSS temporal output'\n",
+    "    \n",
+    "    assert variable_values_all_in_range(hoss_temporal_file_name, '/time', 720.0, 900.0), 'Temporal dimension not correctly subsetted'\n",
+    "\n",
+    "    print_success('HOSS temporal request.')\n",
+    "else:\n",
+    "    print(f'HOSS is not configured for environment: \"{harmony_environment}\" - skipping test.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ee6a0ad",
+   "metadata": {},
+   "source": [
+    "### Named dimension subsetting:\n",
+    "\n",
+    "The following test will recreate a bounding box subset, however, it will explicitly name the dimension variables, rather than relying on the generic bounding box request parameters, `subset=lat(a:b)&subset=lon(c:d)`. For the RSSMIF16D the longitude and latitude dimensions are named:\n",
+    "\n",
+    "* `/latitude` and `/longitude`."
```

Comment:
I was initially going to just reuse the bounding box constraints from an earlier test (bounding box and variable subset), however, that included a minimum latitude of 0, which has unrelated issues with `harmony-py` (see [this PR](https://github.com/nasa/harmony-py/pull/49)).